solr/bin/solr

```diff
@@ -2097,6 +2097,14 @@ else
   SECURITY_MANAGER_OPTS=()
 fi
 
+# Enable ADMIN UI by default, and give the option for users to disable it
+if [ "$SOLR_ADMIN_UI_DISABLED" == "true" ]; then
+  SOLR_ADMIN_UI="-DdisableAdminUI=true"
+  echo -e "ADMIN UI Disabled"
+else
+  SOLR_ADMIN_UI="-DdisableAdminUI=false"
```

Comment:
It doesn't look like this property is ever passed to the run command?

Reply:
You have to explicitly pass it, or you can pass the system property

Reply:
So if I set `SOLR_ADMIN_UI_DISABLED=true` in my `solr.in.sh`, then it sets `SOLR_ADMIN_UI="-DdisableAdminUI=true"` here, and then... does nothing? Because the `SOLR_ADMIN_UI` variable is never read. What am I missing?

Reply:
Yes, that was an error on my part. I forgot to carry that over from the previous PR. I've added it. See line 2219

Reply:
https://github.com/apache/lucene-solr/pull/1471/commits/dedcdfe8b9af18d1a4662937be96b4d710963826 @madrob